### PR TITLE
last updates

### PR DESCRIPTION
**Pull request by sushi2k**
Thank you for submitting a Pull Request to the Mobile Security Testing Guide. Please make sure that:

- [ ] Your contribution is written in the 2nd person (e.g. you)
- [ ] Your contribution is written in an active present form for as much as possible.
- [ ] You have made sure that the reference section is up to date (e.g. please add sources you have used, make sure that the references to MITRE/MASVS/etc. are up to date)
- [ ] Your contribution has proper formatted markdown and/or code
- [ ] Any references to website have been formatted as [TEXT](URL “NAME”)
- [ ] You verified/tested the effectiveness of your contribution (e.g.: is the code really an effective remediation? Please verify it works!)

If your PR is related to an issue. Please end your PR test with the following line:
This PR covers issue #< insert number here >.

### CODE REVIEW

File: Document/0x06b-Basic-Security-Testing.md [hunk 1]
```diff
@@ -100,13 +102,12 @@ The iOS jailbreak scene evolves so rapidly that providing up-to-date instruction
 
 <img src="Images/Chapters/0x06b/cydia.png" alt="iOS App Folder Structure" width="250">
 
-Once you've jailbroken your iOS device and Cydia has been installed (as shown in the previous screenshot), proceed as follows:
+Once you've jailbroken your iOS device and either Cydia (see screenshot above) or Sileo has been installed, you can install the OpenSSH package. Once installed do the following:
```
Comment (TheDauntless) [inline Document/0x06b-Basic-Security-Testing.md]: Maybe add that in the rest of the guide "Cydia" will be used, but Sileo should offer the same packages.

File: Document/0x06b-Basic-Security-Testing.md [hunk 2]
```diff
@@ -116,7 +117,7 @@ You may also want to add the HackYouriPhone repository to get the AppSync packag
 
 You can also easily install Frida by adding the following repository to Cydia <https://build.frida.re>.
 
-The following are some useful packages you can install from Cydia to get started:
+The following are some useful packages you can install from Cydia to get started :
```
Comment (TheDauntless) [inline Document/0x06b-Basic-Security-Testing.md]: Added space by accident?

**Comment by OMTGreleaser** [comment]
Results for commit de73b56a5e2d4f4adf7f1020409686dbff316fba: Broken link result: [!] Error(s) found by the Links validator: 3 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       15 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

File: Document/0x06b-Basic-Security-Testing.md (hunk 2 above)
Comment (sushi2k) [reply Document/0x06b-Basic-Security-Testing.md]: done

File: Document/0x06b-Basic-Security-Testing.md (hunk 1 above)
Comment (sushi2k) [reply Document/0x06b-Basic-Security-Testing.md]: done

**Comment by OMTGreleaser** [comment]
Results for commit ed5d997c0f7f9f4b2e391e40d03017ecd759c8bb: Broken link result: [!] Error(s) found by the Links validator: 2 pages have dead links! Verbose output in /link-check-result.out .
 Please run the Apply_Link_Check.sh in the tools folder to see wherhe the Link issues are. 
 Markdown result errorlength:       15 ../linter-result.out .
 Please run the Apply_Linter_check.sh in the tools folder to see where the Lint issues are.

**Pull request merged**